packages/vm/src/evm/stack.ts

```diff
@@ -80,12 +81,16 @@ export default class Stack {
    * Pushes a copy of an item in the stack.
    * @param position - Index of item to be copied (1-indexed)
    */
-  dup(position: number) {
+  // I would say that we do not need this method any more
+  // since you can't copy a primitive data type
+  // Nevertheless not sure if we "loose" something here?
+  // Will keep commented out for now
```

Comment:
right BigInt is immutable so it can't be changed, but wouldn't we still want to push it onto the stack?